### PR TITLE
Pwr Game can now produce research points when consumed by scientists

### PR DESCRIPTION
**Pull request by deathride58**
Title.

:cl: deathride58
add: Drinking Pwr Game as a scientist will now increase the station's research point generation rate!
add: Unfortunately, though, scientists are now capable of getting drunk off of Pwr Game!
/:cl:

The players want pwrgame&d back. This restores pwrgame&d.

~~The generation rate is 0.5 research points per metabolism cycle, for those too lazy to read the code.~~

### CODE REVIEW

**Comment by ishitbyabullet** [comment]
damn, you fixed techwebs once and for all.

You did it!

**Comment by ShiggyDiggyDo** [comment]
>CARGO ORDER MORE SOFTDRINKS SUPPLY CRATES!!

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -484,10 +484,16 @@
 	glass_icon_state = "glass_red"
 	glass_name = "glass of Pwr Game"
 	glass_desc = "Goes well with a Vlad's salad."
+	var/datum/techweb/linked_techweb
```
Comment (ninjanomnom) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: Don't store it, just grab the techweb in the proc and add points all at once

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Review by Tlaltecuhtli — APPROVED** [review]

**Comment by PKPenguin321** [comment]
i think vapes would make this OP

**Comment by Iamgoofball** [comment]
you should make this get you really really hammered if you're a scientist

**Comment by deathride58** [comment]
@Iamgoofball that is out of this PR's scope.

**Comment by Iamgoofball** [comment]
No it's balance that's funny

Just call it the ballmer peak

**Comment by kevinz000** [comment]
@Iamgoofball is right

**Comment by Iamgoofball** [comment]
just make it work like this:
ballmer_peak

**Comment by PKPenguin321** [comment]
Goof's idea is excellent actually

**Comment by Iamgoofball** [comment]
Will code after my shift tomorrow

**Comment by Cyberboss** [comment]
Ha. Ha.

No

**Comment by Iamgoofball** [comment]
@Cyberboss buzzkill

time 2 redo

**Pull request closed without merging**